### ISSUE TITLE
Add an ability to set infinite reconnect timeout (disable reconnection)

### ISSUE THREAD

**Issue by vyshkant** [Marfusios/websocket-client#8]
Hi!

Awesome lib, thanks! May I ask the authors for the feature which adds an ability to disable reconnection? For example, by adding the property `TimeSpan ReconnectTimeout` and supporting `System.Threading.Timeout.InfiniteTimeSpan`?

```
var client = new WebsocketClient(uri)
{
    ReconnectTimeout= System.Threading.Timeout.InfiniteTimeSpan // this coud disable reconnect
};
```

**Comment by Marfusios** [comment]
Hello, 

thanks for using. Good idea, I will prepare some new property like `bool IsReconnectionEnabled`

**Comment by Marfusios** [comment]
Implemented in 159c68adfdfdc177012f876fb2097de7516b3af2 
There is a new property `IsReconnectionEnabled`
